src/adapters/bidderFactory.js

```diff
@@ -345,6 +347,67 @@ export function newBidder(spec) {
   }
 }
 
+export function preloadBidderMappingFile(fn, adUnits) {
+  let adPodBidders = adUnits
+    .filter((adUnit) => deepAccess(adUnit, 'mediaTypes.video.context') === ADPOD)
```

Comment:
Should we also check if `adpod.brandCategoryExclusion` is true before loading these files? We only need brand category in that case.